### DISCUSSION TITLE
Top Languages Card - Hide Languages C# not working

### DISCUSSION THREAD

**Discussion by melxincognito** [anuraghazra/github-readme-stats#1908] [Q&A] [answered]
Option to hide C# is not working for top languages card

Using <b> csharp</b> in hide
```
<img src='https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,csharp'/>

```
Result: 
<img src='https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,csharp'/>

Using <b> c-sharp</b> in hide
```
<img src='https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,c-sharp'/>

```
Result: 

<img src='https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,c-sharp'/>

Using <b> c#</b> in hide

```
<img src='https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,c#'/>
```

Result

<img src='https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,csharp'/>

**Comment by rickstaa** [comment, marked as answer]
@melxincognito Thanks for your question. As explained in the [README.md](https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options), you should use percent-encoding for symbols included in the language names:

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,c%23)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=melxincognito&layout=compact&theme=radical&hide=swift,c%23)](https://github.com/anuraghazra/github-readme-stats)

**Reply by rickstaa** [reply]
@melxincognito Did that answer your questions?

**Reply by melxincognito** [reply]
Yes! It worked perfectly. Sorry for the delayed response, thanks so much for your help.
